lib/datadog/tracing/contrib/active_record/patcher.rb

```diff
@@ -19,6 +20,10 @@ def target_version
 
           def patch
             Events.subscribe!
+
+            if Integration.version >= Gem::Version.new('7.0.0') && ConcurrentRuby::Integration.patchable?
```

Comment:
I would love to hear if this is the "right" way to leverage a different DD tracing integration. There is a bit of a weird interplay between Active Record and Concurrent Ruby, so would love to hear if this is OK or if there is different preferred approach.